### PR TITLE
Added in About Me section

### PR DESCRIPTION
**Pull request by danicatt**
Completed about me section. Added image, heading and text section.

### CODE REVIEW

File: task-header.css [hunk 1]
```diff
@@ -102,11 +102,38 @@ body {
...
-	text-align: center;
+.aboutme {
+	display: flex;
+	align-items:row;
+	border-top: 4px solid rgb(87, 118, 128);
+	width: 100%;
+	height: 300px;
+	margin: 0;
+	background-color: rgb(130, 157, 178);
+	}
```
Comment (samcrudge) [inline task-header.css:113]: why have you indented this inline with elements?

File: task-header.html [hunk 2]
```diff
@@ -23,14 +23,16 @@
...
+			</div>	
 		</header>
 		
-		<div><h3 class="abouttitle">About Me<h3></div>
-		
-		<div class="aboutme">
-			<img class=>
-		</div>
+		<section class="aboutme">
+				<div><img src="smiley-2055681_1920.jpg" img class="aboutmepic"></div>
```
Comment (samcrudge) [inline task-header.html:30]: for readability possibly?
```suggestion
				<div>
				           <img src="smiley-2055681_1920.jpg" img class="aboutmepic">
				 </div>	
```

File: task-header.html [hunk 3]
```diff
@@ -23,14 +23,16 @@
...
 		</header>
 		
-		<div><h3 class="abouttitle">About Me<h3></div>
-		
-		<div class="aboutme">
-			<img class=>
-		</div>
+		<section class="aboutme">
+				<div><img src="smiley-2055681_1920.jpg" img class="aboutmepic"></div>	
+				<div class="aboutmetext">
```
Comment (samcrudge) [inline task-header.html:31]: indentation possibly here?

**Review by samcrudge — CHANGES_REQUESTED** [review]

File: task-header.css (hunk 1 above)
Comment (danicatt) [reply task-header.css:113]: It was just my preference for readability. But can see on Github, it doesn't read as well - I've pushed the indentation back.

File: task-header.html (hunk 2 above)
Comment (danicatt) [reply task-header.html:30]: Yeah :)

File: task-header.html (hunk 3 above)
Comment (danicatt) [reply task-header.html:31]: Good spot, I've pushed the div back left a bit so it now lines up.

**Review by danicatt — COMMENTED** [review]
Updated indentation to improve readbility.

**Review by samcrudge — APPROVED** [review]
Looks clean!

**Pull request merged**